src/runtime/resource/detail/detail_partitioner.cpp

```diff
@@ -225,6 +225,18 @@ namespace hpx { namespace resource { namespace detail
             throw_runtime_error("partitioner::partitioner",
                 "Cannot instantiate more than one resource partitioner");
         }
+	
+        if(HPX_HAVE_MAX_CPU_COUNT < topo_.get_number_of_pus())
+	{
+                
+		throw_runtime_error(
+                "partitioner::partioner",
+		"HPX_HAVE_MAX_CPU_COUNT is set as " +
+                std::to_string(HPX_HAVE_MAX_CPU_COUNT) +
+                " your system has " +
+                std::to_string(topo_.get_number_of_pus()) +
+                ". Please recompile HPX and increase the maximal CPU count!");
```

Comment:
This should point to the right cmake variable or better yet, doc section.

Reply:
I'd suggest to change the error message to:
```
Currently, HPX_HAVE_MAX_CPU_COUNT is set to 64 while your system has 160 
processing units. Please reconfigure HPX with -DHPX_WITH_MAX_CPU_COUNT=160 
(or higher) to increase the maximal CPU count.
```

Reply:
Sure, I will update the error message